src/main/groovy/org/shipkit/internal/notes/contributors/Contributors.java

```diff
@@ -10,11 +10,12 @@
     /**
      * Fetches contributors from GitHub. Needs GitHub auth token.
      *
+     * @param apiUrl address of GitHub api endpoint, for example: "https://api.github.com/"
      * @param repository name of GitHub repository, for example: "mockito/mockito"
```

Comment:
As in the comment above - if we set URL without slash at the end, we shouldn't suggest to do so in javadoc.